core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -0,0 +1,767 @@
+      Category.USER_ERROR, "0005", "The limit cannot be negative. Operation: %s", "", ""),
+  OPERATION_CHECK_ERROR_CROSS_PARTITION_SCAN(
+      Category.USER_ERROR, "0006", "Cross-partition scan is not enabled. Operation: %s", "", ""),
+  OPERATION_CHECK_ERROR_CROSS_PARTITION_SCAN_ORDERING(
+      Category.USER_ERROR,
+      "0007",
+      "Cross-partition scan ordering is not enabled. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_CROSS_PARTITION_SCAN_FILTERING(
+      Category.USER_ERROR,
+      "0008",
+      "Cross-partition scan filtering is not enabled. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_PROJECTION(
+      Category.USER_ERROR,
+      "0009",
+      "The specified projection is not found. Projection: %s, Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_CLUSTERING_KEY_BOUNDARY(
+      Category.USER_ERROR,
+      "0010",
+      "The clustering key boundary is not properly specified. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_START_CLUSTERING_KEY(
+      Category.USER_ERROR,
+      "0011",
+      "The start clustering key is not properly specified. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_END_CLUSTERING_KEY(
+      Category.USER_ERROR,
+      "0012",
+      "The end clustering key is not properly specified. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_ORDERING_NOT_PROPERLY_SPECIFIED(
+      Category.USER_ERROR, "0013", "Orderings are not properly specified. Operation: %s", "", ""),
+  OPERATION_CHECK_ERROR_ORDERING_COLUMN_NOT_FOUND(
+      Category.USER_ERROR,
+      "0014",
+      "The specified ordering column is not found. Ordering: %s, Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_CONDITION(
+      Category.USER_ERROR,
+      "0015",
+      "The condition is not properly specified. Operation: %s",
+      "",
+      ""),
+  TABLE_NOT_FOUND(Category.USER_ERROR, "0016", "Table does not exist. Table: %s", "", ""),
+  OPERATION_CHECK_ERROR_INVALID_COLUMN(
+      Category.USER_ERROR,
+      "0017",
+      "The column value is not properly specified. Column: %s, Operation: %s",
+      "",
+      ""),
+  EMPTY_MUTATIONS_SPECIFIED(Category.USER_ERROR, "0018", "The mutations are empty", "", ""),
+  OPERATION_CHECK_ERROR_MULTI_PARTITION_MUTATION(
+      Category.USER_ERROR,
+      "0019",
+      "Mutations that span multiple partitions are not supported. Mutations: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_PARTITION_KEY(
+      Category.USER_ERROR,
+      "0020",
+      "The partition key is not properly specified. Operation: %s",
+      "",
+      ""),
+  OPERATION_CHECK_ERROR_CLUSTERING_KEY(
+      Category.USER_ERROR,
+      "0021",
+      "The clustering key is not properly specified. Operation: %s",
+      "",
+      ""),
+  NOT_SUPPORTED_IN_COMMUNITY_EDITION(
+      Category.USER_ERROR, "0022", "This feature is not support in the ScalarDB Community edition", "", ""),
```

Comment:
Sorry, I had a misspelling in my previous comment about this message.

```suggestion
      Category.USER_ERROR, "0022", "This feature is not supported in the ScalarDB Community edition", "", ""),
```